### PR TITLE
Application Commands Checks

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

File: DisCatSharp.ApplicationCommands/Workers/ApplicationCommandEqualityChecks.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+{
+	internal static class ApplicationCommandEqualityChecks
+	{
+		public static bool IsEqualTo(this DiscordApplicationCommand ac1, DiscordApplicationCommand targetApplicationCommand)
+		{
+			if (ac1 == null && targetApplicationCommand == null)
+				return false;
+
+			if (targetApplicationCommand == null)
+				return false;
```
Comment (JBraunsmaJr) [inline DisCatSharp.ApplicationCommands/Workers/ApplicationCommandEqualityChecks.cs:39]: Can combine these

**Review by JBraunsmaJr — CHANGES_REQUESTED** [review]
Looking pretty good, can combine the one thing and merge

File: DisCatSharp.ApplicationCommands/Workers/ApplicationCommandEqualityChecks.cs (hunk 1 above)
Comment (Lulalaby) [reply DisCatSharp.ApplicationCommands/Workers/ApplicationCommandEqualityChecks.cs:39]: yus will do

**Comment by KazutoKashima** [comment]
Lgtm

**Review by villChurch — APPROVED** [review]
Looks good to me

**Pull request merged**